src/main/java/uk/gov/hmcts/cmc/claimstore/services/notifications/ClaimIssuedNotificationService.java

```diff
@@ -78,28 +79,32 @@ public void logNotificationFailure(final NotificationException exception,
         ImmutableMap.Builder<String, String> parameters = new ImmutableMap.Builder<>();
         parameters.put(CLAIM_REFERENCE_NUMBER, claim.getReferenceNumber());
         parameters.put(CLAIMANT_NAME, getNameWithTitle(claim.getClaimData().getClaimant()));
-
-        if (!claim.getClaimData().isClaimantRepresented()) {
```

Comment:
Hey @dharmendrak , @sabahirfan or @robertparkinson , are you using this notification as well? If so I think it will break your app in case there is more then one defendant. Please check it, I am not sure why @comanirudha removed this condition.